### ISSUE TITLE
List of files covered by an overlay

### ISSUE THREAD

**Issue by runeflobakk** [mystilleef/eclipse4-fastopen#2]
(new issue created from https://github.com/mystilleef/eclipse4-fastopen/issues/1#issuecomment-102982604 for reference)

I see files in the FastOpen-dialog, but they are truncated, and I can't move what looks like another column on top of the files. See screennshot:

![screen shot 2015-05-18 at 12 04 57](https://cloud.githubusercontent.com/assets/174823/7678657/29e177d4-fd56-11e4-9a27-04b70698d652.png)

Testing with the update-site proposed in https://github.com/mystilleef/eclipse4-fastopen/issues/1#issuecomment-103310474 solves the problem.

**Comment by mystilleef** [comment]
Can you test to see if the other plugin conflict problem is solved too?

**Comment by runeflobakk** [comment]
Yes, it seems to be solved :+1: 
I reinstalled the buggy version of [ZipEditor](http://sourceforge.net/projects/zipeditor/), and I don't get any exceptions in the log. (though zip-files does not show up in FastOpen's list, but I assume that is intentional). 

Found a new version of the ZipEditor-plugin in the [NodeClipse](http://www.nodeclipse.org/) plugin suite, and ZipEditor seems to be fixed there. No exceptions in the log, and zip-files show up in the FastOpen file list.
